src/components/SchedulePage.js

```diff
@@ -79,72 +74,79 @@ export default function SchedulePage() {
-  result.setDate(result.getDate() + days);
-  return result;
-}
+  function addDays(date, days) {
+    var result = new Date(date);
+    result.setDate(result.getDate() + days);
+    return result;
+  }
 
-const [dateString, setDateString] = useState(dateMaker(0));
+  const customHeaderStyle = {
+    backgroundColor: '#6C6FA5',
+    color: '#ffffff'
+  } 
 
   return (
     <MaterialTable
       icons={tableIcons}
-      title={'Employee Schedule for week of: ' + dateString}
+      title={'Employee Schedule for week of: ' + mondayToDateRangeString(monday)}
       columns={[
-        { title: 'Employee', field: 'Employee' },
-        { title: 'Monday', field: 'Monday'},
-        { title: 'Tuesday', field: 'Tuesday'},
-        { title: 'Wednesday', field: 'Wednesday'},
-        { title: 'Thursday', field: 'Thursday'},
-        { title: 'Friday', field: 'Friday'},
-        { title: 'Saturday', field: 'Saturday'},
-        { title: 'Sunday', field: 'Sunday'}
+        { title: 'Employee', field: 'Employee', headerStyle: customHeaderStyle},
+        { title: 'Monday', field: 'Monday', headerStyle: customHeaderStyle },
+        { title: 'Tuesday', field: 'Tuesday', headerStyle: customHeaderStyle },
+        { title: 'Wednesday', field: 'Wednesday', headerStyle: customHeaderStyle },
+        { title: 'Thursday', field: 'Thursday', headerStyle: customHeaderStyle },
+        { title: 'Friday', field: 'Friday', headerStyle: customHeaderStyle },
+        { title: 'Saturday', field: 'Saturday', headerStyle: customHeaderStyle },
+        { title: 'Sunday', field: 'Sunday', headerStyle: customHeaderStyle }
       ]}
       data={employeeSchedule}
       isLoading={employeeSchedule.length === 0}
       components={{
         Actions: props => (
           <React.Fragment>
-          <Button
-            onClick={(event) => setOpen(true)}
-            color="primary"
-            variant="contained"
-            style={{textTransform: 'none', marginRight: 15, marginLeft: 15}}
-            size="small"
-          >
-            Change Week
-          </Button>
-          <Dialog onClose={() => setOpen(false)} open={open}>
-          <DialogTitle>Select Week to change</DialogTitle>
-          <List>
-            {[0,1,2,3,4].map((weeksOffset) => (
-              <ListItem button onClick={() => {setDateString(dateMaker(weeksOffset)); refreshSchedule(mondayMaker(weeksOffset)); setOpen(false)}} key={weeksOffset}>
-                <ListItemText primary={dateMaker(weeksOffset)} />
-              </ListItem>
-            ))}
-          </List>
-          </Dialog>
+            <div  style={{display: "flex"}}>
+              <Button
+                onClick={() => {
+                  let previousMonday = addDays(monday, -7);
+                  setMonday(previousMonday);
+                  refreshSchedule(previousMonday);
+                }}
+                color="primary"
+                variant="contained"
+                style={{textTransform: 'none'}}
```

Comment:
```suggestion
                style={{textTransform: 'none', marginRight: 10}}
```
Just so the buttons don't seem attached. Looks Great!